### PR TITLE
docs: sync documents

### PR DESCRIPTION
**Pull request by misitebao**
- Sync Chinese and Japanese documents.

### CODE REVIEW

File: website/i18n/ja/docusaurus-plugin-content-docs/current/guides/migrating.mdx [hunk 1]
```diff
@@ -170,21 +170,21 @@ The format of the file is slightly different. Here is a comparison:
...
-| frontend / dir     |                  | Removed                                             |
-| frontend / install | frontend:install | Changed                                             |
-| frontend / build   | frontend:build   | Changed                                             |
-| frontend / bridge  |                  | Removed                                             |
-| frontend / serve   |                  | Removed                                             |
-| tags               |                  | Removed                                             |
-|                    | wailsjsdir       | The directory to generate wailsjs modules           |
+| v1                 | v2               | Notes                                                                                                                                                                      |
+| ------------------ | ---------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
Comment (leaanthony) [inline website/i18n/ja/docusaurus-plugin-content-docs/current/guides/migrating.mdx:174]: What's this enormous line? :-)

File: website/i18n/zh-Hans/docusaurus-plugin-content-pages/changelog.mdx [hunk 2]
```diff
@@ -4,14 +4,21 @@
...
+- `Security` 对安全性的改进。
+
+## [即将发布]
+
+### 新增
+- 添加 `OpenInspectorOnStartup` 到调试选项以允许在调试模式下启动应用程序期间打开 Web 检查器。 由 @stffabi 在这个 [PR](https://github.com/wailsapp/wails/pull/2080) 中添加。
+- 在 macOS 上的 `wails doctor`，现在也会显示安装的 Xcode 版本。 由 @stffabi 在这个 [PR](https://github.com/wailsapp/wails/pull/2089) 中添加。
+
+### 修复
+- wails dev 中 `noreredload` 标志未被应用。 由 @stffabi 在这个 [PR](https://github.com/wailsapp/wails/pull/2081) 中修复。
```
Comment (leaanthony) [inline website/i18n/zh-Hans/docusaurus-plugin-content-pages/changelog.mdx:21]: typo in `noreload` 👍

**Review by leaanthony — APPROVED** [review]
Just a couple of points 👍

File: website/i18n/ja/docusaurus-plugin-content-docs/current/guides/migrating.mdx (hunk 1 above)
Comment (misitebao) [reply website/i18n/ja/docusaurus-plugin-content-docs/current/guides/migrating.mdx:174]: This is the completion format generated by Crowdin and does not affect rendering.

File: website/i18n/zh-Hans/docusaurus-plugin-content-pages/changelog.mdx (hunk 2 above)
Comment (misitebao) [reply website/i18n/zh-Hans/docusaurus-plugin-content-pages/changelog.mdx:21]: Fixed ^_^

**Pull request merged**